test/utils/text_display.jl

```diff
@@ -1,7 +1,36 @@
+using ADRIA: human_readable_name, get_scientific_factors
+
 @testset "text_display" begin
-    @testset "get_scientific_factors" begin
-        get_scientific_factors = ADRIA.get_scientific_factors
+    @testset "human_readable_name" begin
+        @testset "Symbols inputs" begin
```

Comment:
Remove double plural

```suggestion
        @testset "Symbol inputs" begin
```